.pylintrc

```diff
@@ -0,0 +1,8 @@
+[FORMAT]
+max-line-length=120
+
+[BASIC]
+good-names=
+    e,
+    f,
+    py
```

Comment:
Why don't you include this configuration in `pyproject.toml`?

Reply:
I have no preference, the config can be in any of these planes, in `pyproject.toml` or in `.pylintrc`. What is the advantage of using pyproject.toml?

Reply:
I prefer having all configuration in one place (pyproject.toml). You don't have to remember and look for a multiple config files. Often configuration of dev tools has to match (like max line length). This is easier to take into account when the config isn't spread over multiple files. Follow up: #26

Reply:
ok, thank you for following up on this